scipy/interpolate/tests/test_rgi.py

```diff
@@ -594,6 +594,33 @@ def test_nonscalar_values_linear_2D(self):
         v2 = np.expand_dims(vs, axis=0)
         assert_allclose(v, v2, atol=1e-14, err_msg=method)
 
+    @pytest.mark.parametrize("dtype",
+            [np.float32, np.float64, np.complex64, np.complex128])
```

Comment:
This should fix your linter error

```suggestion
    @pytest.mark.parametrize(
        "dtype",
        [np.float32, np.float64, np.complex64, np.complex128]
    )
```

Reply:
Thanks Kai